base/src/main/scala/co/uproot/abandon/Ast.scala

```diff
@@ -369,4 +371,9 @@ case class Scope(entries: Seq[ASTEntry], parentOpt: Option[Scope]) extends ASTEn
         includedScopes.foreach(_.checkDupes())
     }
   }
+
+  def checkUnusedSymbols() {
+    definitions.filterNot(_.isUsed)
+      .foreach(d => println(s"symbol `${d.name}` defined in ${d.pos} but never used"))
```

Comment:
As far as I've seen all errors are written in stdout using println, except one occasion in UI. But there is one print out using formatting for the printed string:

`println(Console.RED + Console.BOLD + msg + Console.RESET)`

I think that's a good way to print a warning, but maybe using another colour?